src/blueapi/service/runner.py

```diff
@@ -141,7 +140,7 @@ def _rpc(
     **kwargs: Any,
 ) -> T:
     mod = import_module(module_name)
-    func: Callable[P, T] = _validate_function(
+    func: Callable[..., T] = _validate_function(
```

Comment:
Should: This is less type safe since mypy/pyright won't check the function parameters, see [docs](https://sobolevn.me/2021/12/paramspec-guide), please revert.

Reply:
there was an error in the IDE

Reply:
What was the error? Did you understand what was causing it?

Reply:
the typing system was not happy, can look into what exactly it is